CHANGELOG.rst

```diff
@@ -1,6 +1,44 @@
 Changelog
 =========
 
+1.10.0 (2022-09-29)
+------------------
+
+New Operators
+"""""""""""""
+
+This release adds the following 2 new async sensors/operators:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operator/Sensor Class
+     - Import Path
+     - Example DAG
+
+   * - ``ExternalDeploymentTaskSensorAsync``
+     - .. code-block:: python
+
+        from astronomer.providers.http.sensors.external_deployment_task import ExternalDeploymentTaskSensorAsync
```

Comment:
We should move it to https://github.com/astronomer/astronomer-providers/tree/main/astronomer/providers/core/sensors

Reply:
That should be in separate PR

Reply:
yes sure will create a separate PR for this and move to core sensors.

Reply:
@bharanidharan14  can you move this sensor to core?

Reply:
I have created it at https://github.com/astronomer/astronomer-providers/pull/684

Reply:
PR https://github.com/astronomer/astronomer-providers/pull/684 got merged.